firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -165,7 +165,8 @@ public InstallationResponse createFirebaseInstallation(
           return readCreateResponse(httpURLConnection);
         }
 
-        logFisCommunicationError(httpURLConnection);
+        logFisCommunicationError(
+            httpURLConnection, availableFirebaseOptions(appId, apiKey, projectID));
```

Comment:
rather than repeating `availableFirebaseOptions(` 3 times, I recommend calling `#availableFirebaseOptions` from within the `#logFisCommunicationError` method.